Automattic/wp-calypso
# Components: Output type defs

#### Changes proposed in this Pull Request

Output type definitions generated by `tsc` on the components package.

#### Testing instructions

The following should not error and list some type definitions:

```sh
npx lerna run clean --scope='*/components' && npx lerna run prepare --scope='*/components' && ls packages/components/types
```

The following no longer lists missing type definitions for the `@automattic/components` package (after the above command builds the typedefs, part of `prepare` script):

```sh
npm run typecheck -- -p client/landing/gutenboarding
```
